eos/b-decays/b-to-d-l-nu.cc

```diff
@@ -89,48 +140,87 @@ namespace eos
-            double norm = 8.0 * v2 * m_B * s * power_of<2>(g_fermi())
-                / (3.0 * 256.0 * power_of<3>(M_PI * m_B));
-            // helicity amplitudes
-            const double hhV0 = 2.0 * m_B * p * fp / (sqrt(s));
-            const double hhVt = (m_B * m_B - m_D * m_D) * f0 / (sqrt(s));
-            const double hhS  = (m_B * m_B - m_D * m_D) * f0 / (mbatmu - mcatmu);
-            const double hhT  = -2.0 * m_B * p * fT / (m_B + m_D);
-            // NP contributions in EFT including tensor operator, cf. e.g. Sakaki:2013bfa
+            double ml_hat = sqrt(1.0 - v);
+            double nD = v2 * m_B * s * power_of<2>(g_fermi()) / (256.0 * power_of<3>(M_PI * m_B));
+            // NP contributions in EFT including tensor operator (cf. [DSD2014]). (in SM cvl=1, and all other couplings are zero)
             const WilsonCoefficients<BToC> wc = model->wilson_coefficients_b_to_c(opt_l.value(), false);
-            const complex<double> cv1 = wc.cvl() - 1.0;
-            const complex<double> cv2 = wc.cvr();
-            const complex<double> cs1 = wc.csr();
-            const complex<double> cs2 = wc.csl();
-            const complex<double> cT  = wc.ct();
-            // normalized(|V_cb|=1) differential decay width including NP (in SM cvl=1, and all other couplings are zero)
-            return norm * p * (
-                               std::norm(1.0 + cv1 + cv2) * ((1.0 + mvm1 / 2.0) * power_of<2>(hhV0) + (3.0 * mvm1 / 2.0) * power_of<2>(hhVt))
-                               + 3.0 / 2.0 * std::norm(cs1 + cs2) * power_of<2>(hhS) + 8.0 * std::norm(cT) * (1.0 + 2.0 * mvm1) * power_of<2>(hhT)
-                               + 3.0 * std::real((1.0 + cv1 + cv2) * (std::conj(cs1) + std::conj(cs2))) * sqrt(mvm1) * hhS * hhVt
-                               - 12.0 * std::real((1.0 + cv1 + cv2) * std::conj(cT)) * sqrt(mvm1) * hhT * hhV0
-                               );
+            const complex<double> vl = wc.cvl() - 1.0;
+            const complex<double> vr = wc.cvr();
+            const complex<double> sl = wc.csl();
+            const complex<double> sr = wc.csr();
+            const complex<double> gV = vr + vl;
+            const complex<double> gA = vr - vl;
+            const complex<double> gS = sr + sl;
+            const complex<double> gP = sr - sl;
+            const complex<double> tl = wc.ct();
+            // helicity amplitudes, cf. [DSD2014] eqs. 13-14
+            const complex<double> hh0 = 2.0 * m_B * p * fp * (1.0 + gV) / sqrt(s) ;
+            const complex<double> hht = (1.0 + gV) * (m_B * m_B - m_D * m_D) * f0 / sqrt(s);
+            const complex<double> hhS = - gS * (m_B * m_B - m_D * m_D) * f0 / (mbatmu - mcatmu);
+            const complex<double> hhT = - 2.0 * m_B * p * fT * tl / (m_B + m_D);
+            const complex<double> hhtS = hht - hhS / ml_hat;
+
+            return 4.0 * nD * p * ( std::abs(hh0) * std::abs(hhtS) * (1.0 - v) - 4.0 * ml_hat * std::real(hhT * std::conj(hhtS)) );
```

Comment:
Should'nt there be a minus sign in front of

  std::abs(hh0) * std::abs(hhtS) * (1.0 - v)

???

It comes from the term in (eq.12)

  (|H0| cosTh - |Hts| )^2 = |H0|^2 cosTh^2 - 2 |H0| |Hts| cosTh + |Hts|^2

and in the AFB only the one ~ cosTh enters

Reply:
Maybe we could agree how you define the A_FB:

 A_FB = int_0^1 d^2Gamma - int_-1^0 d^2Gamma

is what I am used to. Then you are missing an overall minus sign, i.e.:

  4.0 * nD * p * ...   ->  - 4.0 * nD * p * ...

If you define 

  A_FB = int_-1^0 d^2Gamma - int_0^1 d^2Gamma

then it is ok, but for me this is an unusual definition.